src/shapes/space_filler/fillers.js

```diff
@@ -0,0 +1,87 @@
+// L-system instructions for space filling curves
+
+export const fillers = {
+  // http://mathforum.org/advanced/robertd/lsys2d.html
+  'Gosper (flowsnake)': {
```

Comment:
These need to match exactly the keys in the option. Should we use these keys() to generate the options?

Reply:
I didn't because I intend to create another non-eraser shape that uses all kind of L-system patterns and I was going to include them in this list. But I think you are right - I can have 2 different lists. I'll make this change.